shell/platform/windows/direct_manipulation.cc

```diff
@@ -53,16 +56,31 @@ HRESULT DirectManipulationEventHandler::OnViewportStatusChanged(
             (int32_t) reinterpret_cast<int64_t>(this));
       }
     }
-  } else if (previous == DIRECTMANIPULATION_RUNNING) {
+  }
+  if (previous == DIRECTMANIPULATION_RUNNING) {
```

Comment:
Can we combine the previous `if` clause and this one into
```suggestion
  if (previous == DIRECTMANIPULATION_RUNNING) {
    if (during_synthesized_reset_) {
      during_synthesized_reset_ = false;
    } else if (current == DIRECTMANIPULATION_RUNNING) {
      ...
    }
    // Reset deltas to ensure only inertia values will be compared later.
    ...
```

(Did I suggest the current way when I first reviewed this file? Sorry if I did.)

Reply:
We can't do that since the `else if` would no longer make any sense (previous and current state will not ever be the same).